inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/AbstractSourceOperator.java

```diff
@@ -155,22 +153,25 @@ public void updateOpt(SourceRequest request, Integer groupStatus, String operato
         entity.setModifier(operator);
 
         // Re-issue task if necessary
-        entity.setPreviousStatus(entity.getStatus());
-        if (GroupStatus.forCode(groupStatus).equals(GroupStatus.CONFIG_SUCCESSFUL)) {
-            entity.setStatus(SourceStatus.TO_BE_ISSUED_ADD.getCode());
-        } else {
-            switch (SourceStatus.forCode(entity.getStatus())) {
-                case SOURCE_NORMAL:
-                    entity.setStatus(SourceStatus.TO_BE_ISSUED_ADD.getCode());
-                    break;
-                case SOURCE_FAILED:
-                    entity.setStatus(SourceStatus.SOURCE_NEW.getCode());
-                    break;
-                default:
-                    // others leave it be
-                    break;
+        if (!entity.getSourceType().equals(SourceType.AUTO_PUSH)) {
```

Comment:
Suggest moving this check to the beginning of this method, if it is `AUTO_PUSH` source, just print a log and then return.